src/ground-commander/ground-commander-main.cpp

```diff
@@ -105,10 +105,10 @@ int read_command()
         // TODO: write to normal pipes
         int data_bytes_written = commander->WriteToDataPipe( cmd_buffer);
         // TODO implement passing size // int data_bytes_written = commander->WriteToDataPipe(result, size);
-
         if (data_bytes_written > 0) 
         {
-            return data_bytes_written;
+            fprintf(stderr, "In read command execution, WriteToDataPipe succeded with %d byte written \n", data_bytes_written);
```

Comment:
I like the more detailed messaging, but can you change this to log with Shakespeare? We should be watching the system logs rather than reading stdout or stderr.

Alternatively, if this is a temporary printout to be removed later, wrap it in #ifdef CS1_DEBUG #endif